### PR TITLE
chore: Remove Express server

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,8 +87,8 @@
     "prettier": "prettier \"**/*.{json,md,js,yml}\"",
     "preversion": "yarn && yarn test && yarn dist",
     "start": "pm2 start",
-    "start:dev": "cross-env NODE_DEBUG=\"@wireapp/*\" ts-node src/index.ts",
-    "test": "ts-node -P tsconfig.jasmine.json node_modules/jasmine/bin/jasmine --config=jasmine.json",
+    "start:dev": "node dist/main.js",
```
Comment (bennyn) [inline package.json:90]: Why not using `ts-node` here?

Comment (bennyn) [reply package.json:90]: And why have you removed the `NODE_DEBUG` setting?

**Comment by lipis** [comment]
> `express` is still imported in `InstanceController.ts`, will that work?

Nest has a dependency so it's being installed.

File: package.json (hunk 1 above)
Comment (lipis) [reply package.json]: it was part of the mass remove.. yeap :)

**Pull request merged**